### ISSUE TITLE
remembering storage location

### ISSUE THREAD

**Issue by bsastregx** [dionmunk/vscode-notes#56]
Hi Dion,

thanks for this great addon. Now I don't have to swtich anymore between vscode and notion. It's great.
Just one suggestion. I clicked the configuration icon, because I didn't remember the storage location, and it opens the folder of my current vscode project. It would be nice to get the actual storage location folder when the windows pops ups. 

Bests, Bruno.

**Comment by dionmunk** [comment]
This is an excellent suggestion. I will add this feature in the next release!

**Comment by nyxb** [comment]
is this dead?

**Comment by dionmunk** [comment]
> is this dead?

No, it isn’t dead. I’ve just been extremely busy with other projects.

**Comment by dionmunk** [comment]
Resolved by https://github.com/dionmunk/vscode-notes/pull/64.
